include/pinocchio/spatial/symmetric3.hpp

```diff
@@ -339,6 +340,16 @@ namespace pinocchio
       m_data += s2.m_data; return *this;
     }
 
+    Symmetric3Tpl operator-(const Symmetric3Tpl & s2) const
+    {
+      return Symmetric3Tpl((m_data-s2.m_data).eval());
```

Comment:
The eval is not needed here.
```suggestion
      return Symmetric3Tpl(m_data-s2.m_data);
```